recovery/files/dataset-list.json

```diff
@@ -8,6 +8,10 @@
   "uc-berkeley-computer-science-courses": "UC Berkeley Computer Science Courses",
   "the-librivox-free-audiobook-collection": "The LibriVox Free Audiobook Collection",
   "folding-at-home-covid": "Folding@home COVID-19 Dataset",
```

Comment:
I have left this one here on purpose (backwards / db compatibility?)

Reply:
reasonable. lets ship this and see what happens.